sql/sql_type.h

```diff
@@ -4113,6 +4113,12 @@ class Type_handler
 
   /*
     create a compact size key part for a sort key
+
+    @param to                   buffer to store value of keypart
+    @param item                 item corresponding to the keypart
```

Comment:
Do we get the value from here? If yes I think it's better to mention this explicitly.